mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulFollowingScreen.kt

```diff
@@ -23,6 +27,8 @@ private data class ProfileAdapter(
     get() = profile.name
   override val emoji: String
     get() = profile.emoji
+  override val uid: String
+    get() = profile.uid
```

Comment:
It's not your part, but the doc is missing, We don't understand why we need an adapter here.

You don't need to fix it, I create an issue (#136). If someone knows the person that write this code, add him as assignee in the issue Thx !

Reply:
@Fouad-sys has added some documentation, so I'll link #136 to this PR so it gets closed when it's merged.